civis_jupyter_notebooks/assets/extensions/uncommitted_changes.js

```diff
@@ -0,0 +1,32 @@
+define(['jquery'], function($) {
+  var uncommitted_changes = function(elem) {
+    var settings = {
+      url: '/git/uncommitted_changes',
+      type: 'GET',
+      contentType: 'applicaton/json',
+      success: function(data) {
+        if (data.hasOwnProperty('dirty') && data.dirty) {
+          elem.css('display', 'inline-block');
+        } else {
+          elem.css('display', 'none');
```

Comment:
Hrm, I think we should also have an indicator when the changes are committed. Check with Annie?

Reply:
i'll check with her. but how would we distinguish between a user recently committing something and there being nothing for them to commit. We'd have to plug into the shell channel to capture any `git commit` commands being written.